forms-flow-web/src/components/ServiceFlow/details/TaskHeaderListView.js

```diff
@@ -0,0 +1,364 @@
+              );
+            }
+           
+          } else {
+            dispatch(setBPMTaskDetailUpdating(false));
+          }
+        })
+      );
+    }
+  };
+
+  const onUnClaimTask = () => {
+    dispatch(setBPMTaskDetailUpdating(true));
+    dispatch(
+      // eslint-disable-next-line no-unused-vars
+      unClaimBPMTask(taskId, (err, response) => {
+        if (!err) {
+          if (!SocketIOService.isConnected()) {
+            if (selectedFilter) {
+              dispatch(getBPMTaskDetail(taskId));
+            }
+          }
+          if(selectedFilter){
+            dispatch(
+              fetchServiceTaskList(reqData,null,firstResult)
+            );
+          }
+          
+        } else {
+          dispatch(setBPMTaskDetailUpdating(false));
+        }
+      })
+    );
+  };
+
+  const onFollowUpDateUpdate = (followUpDate) => {
+    setFollowUpDate(followUpDate);
+    dispatch(setBPMTaskDetailUpdating(true));
+    const updatedTask = {
+      ...task,
+      ...{ followUp: followUpDate ? getISODateTime(followUpDate) : null },
+    };
+    dispatch(
+      // eslint-disable-next-line no-unused-vars
+      updateBPMTask(taskId, updatedTask, (err, response) => {
+        if (!err) {
+          if (!SocketIOService.isConnected()) {
+            dispatch(getBPMTaskDetail(taskId));
+            dispatch(
+              fetchServiceTaskList(reqData,null,firstResult)
+            );
+          }
+        } else {
+          dispatch(setBPMTaskDetailUpdating(false));
+        }
+      })
+    );
+  };
+
+  const onDueDateUpdate = (dueDate) => {
+    setDueDate(dueDate);
+    dispatch(setBPMTaskDetailUpdating(true));
+    const updatedTask = {
+      ...task,
+      ...{ due: dueDate ? getISODateTime(dueDate) : null },
+    };
+    dispatch(
+      // eslint-disable-next-line no-unused-vars
+      updateBPMTask(taskId, updatedTask, (err, response) => {
+        if (!err) {
+          if (!SocketIOService.isConnected()) {
+            dispatch(getBPMTaskDetail(taskId));
+            dispatch(
+              fetchServiceTaskList(reqData,null,firstResult)
+            );
+          }
+        } else {
+          dispatch(setBPMTaskDetailUpdating(false));
+        }
+      })
+    );
+  };
+
+  // eslint-disable-next-line no-unused-vars
+  const FollowUpDateInput = React.forwardRef(({ value, onClick }, ref) => {
+    return (
+      <div onClick={onClick} ref={ref}>
+        <i className="fa fa-calendar mr-1" />{" "}
+        {followUpDate ? (
+          <span className="mr-4">{moment(followUpDate).fromNow()}</span>
+        ) : (
+          t("Add Date")
+        )}
+      </div>
+    );
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const DueDateInput = React.forwardRef(({ value, onClick }, ref) => {
+    return (
+      <div onClick={onClick} ref={ref}>
+        <i className="fa fa-calendar mr-1" />{" "}
+        {dueDate ? (
+          <span className="mr-4">{moment(dueDate).fromNow()}</span>
```

Comment:
please check here due date is converting local date but we have already moment function in service micro-front end

Reply:
These time zone issues will change after testing need to verify from testing team